pcsx2/Patch.cpp

```diff
@@ -225,11 +216,13 @@ int LoadPatchesFromZip(wxString gameCRC, const wxString& patchesArchiveFilename)
 	{
 		wxString name = entry->GetName();
 		name.MakeUpper();
-		if (name.Find(gameCRC) == 0 && name.Find(L".PNACH")+6u == name.Length()) {
+		if (name.Find(gameCRC) == 0 && name.Find(L".PNACH") + 6u == name.Length())
+		{
 			PatchesCon->WriteLn(Color_Green, L"Loading patch '%s' from archive '%s'",
-				WX_STR(entry->GetName()), WX_STR(patchesArchiveFilename));
+								WX_STR(entry->GetName()), WX_STR(patchesArchiveFilename));
```

Comment:
Looks like your formatter added extra spacing?

Reply:
Yeah, this is clang's output